CogNative/backend/translation/translation.py

```diff
@@ -0,0 +1,15 @@
+from googletrans import Translator
+
+
+class translation:
+
+    def __int__(self):

```

Comment:
__int__()  ->  __init__()

without this correction, the class won't instantiate self.translator...